web_responsive/views/web.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2016 LasLabs Inc.
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+
+<openerp>
+<data>
+
+    <template id="webclient_bootstrap"
+              inherit_id="web.webclient_bootstrap"
+              name="App Drawer - Web Client"
+              >
+
+        <xpath expr="//td[@class='oe_leftbar']" position="attributes">
+            <field name="style">display: none;</field>
```

Comment:
I'll be pulling in changes we came up with on #446 - so we'll be good here when I do that 😉